n5k/calculator_base.py

```diff
@@ -48,7 +48,9 @@ def get_tracer_parameters(self):
                         1.607983, 1.689579, 1.772899,
                         1.857700, 1.943754, 2.030887,
                         2.118943])
-        return {'b_g': b_g}
+        A_IA = np.full(5, 0.15)	
```

Comment:
do you need this at all? I removed it when merging your branch and everything worked well.

Reply:
Reverted!

Reply:
Thanks a lot for the changes @schoeneberg . I don't see this one in the current commit though.

Reply:
I _hope_ this is fixed now. Let me know if there's any problem after all. In particular, my version of the file now looks like master as far as I am aware

Reply:
Yep, all good!